### PR TITLE
Add aliases to S139 and S201

### PR DESCRIPTION
**Pull request by danflapjax**
At #818, some alternate names for [S139](https://topology.pi-base.org/spaces/S000139) and [S201](https://topology.pi-base.org/spaces/S000201) were mentioned, and I figured it would be good to add them. I also added a few words to each description about how one has a finer topology than the other.

### CODE REVIEW

File: spaces/S000201/README.md
Comment (prabau) [inline spaces/S000201/README.md:6]: This is a sloppy name, not appropriate here, as this is *not* what is normally understood as a wedge product.

File: spaces/S000139/README.md
Comment (prabau) [inline spaces/S000139/README.md:7]: That's undetermined.  How about "Rose with countably many petals"  (no need to say "countably infinitely many")

File: spaces/S000201/README.md
Comment (StevenClontz) [reply spaces/S000201/README.md:6]: I wouldn't use the language "sloppy" myself, but agree that [wedge](https://en.wikipedia.org/wiki/Wedge_sum) can be misleading here and would suggest not including this alias unless it appears in a textbook or widely referenced paper.

File: spaces/S000139/README.md
Comment (StevenClontz) [reply spaces/S000139/README.md:7]: Perhaps "Infinite rose with countably-many petals"? See also [Rose (topology) on Wikipedia](https://en.wikipedia.org/wiki/Rose_(topology)).

Comment (prabau) [reply spaces/S000139/README.md:7]: Could be iffy, as each "rose" is an infinite set already.
But thinking about it some more, I now think that "Rose with infinitely many petals" is fine.  It is not a completely determined description, but that does not matter.  It's only an alias anyway.

File: spaces/S000201/README.md
Comment (prabau) [reply spaces/S000201/README.md:6]: I absolutely did not mean that @danflapjax was sloppy here. :-) I did mean that whoever author used that name was.

Comment (GeoffreySangston) [reply spaces/S000201/README.md:6]: I think the pragmatic case for its inclusion is that it appears as "Example 1.25: The Shrinking Wedge of Circles." in Hatcher's [Algebraic Topology](https://pi.math.cornell.edu/~hatcher/AT/AT.pdf), and (I'm pretty sure) this is the most popular introductory algebraic topology book. 

Here are the stats from Google Scholar:
- ["shrinking wedge of circles"](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C9&q=%22shrinking+wedge+of+circles%22&btnG=)
   16 results 
- ["infinite earring"](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C9&q=%22infinite+earring%22&btnG=)
   12 results
- ["hawaiian earring"](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C9&q=%22hawaiian+earring%22&btnG=&oq=%22hawa)
    698 results

I didn't look very closely, but I saw that the textbook [Basic Algebraic Topology and its Applications](https://link.springer.com/book/10.1007/978-81-322-2843-1) refers to it as "Infinite earring or shrinking wedge of circles". 

>  sloppy and misleading name, not appropriate here, as this is not what is normally understood as a wedge product.

> [wedge](https://en.wikipedia.org/wiki/Wedge_sum) can be misleading here

I actually agree, but a counterpoint is that that use of 'wedge' here demands the learner to learn something more about the wedge product, by drawing their attention to a space for comparison. Naming it so begs the question of why the shrinking wedge of circles is not a wedge sum.

Anyways, I felt the need to counter-argue just because I'm the one who suggested the alias. I did not know that there is a controversy surrounding the name when I brought up the Hawaiian earring space. I have no special interest in this, though after reading the various arguments I've come to believe that the 'Hawaiian' variant is unfortunate and misleading.

Comment (prabau) [reply spaces/S000201/README.md:6]: Actually, I still think the name Hawaiian earring would be preferable as the primary name, as it is the least ambiguous and the canonical name used in the literature.  Opinions may differ.

Comment (prabau) [reply spaces/S000201/README.md:6]: Regarding the proposed alias here, we could decide to not add it as an alias, but instead add a paragraph in the text.  Maybe something like "Appears as Example 1.25 in {{Hatcher}} under the name "Shrinking wedge of circles", which is a misleading name as it is not a wedge sum of circles".

Comment (StevenClontz) [reply spaces/S000201/README.md:6]: I'd lean towards adding the "wedge" alias since it is something a user might see in the textbook and attempt to look up in pi-Base.

I agree with @prabau that we should clarify in our description that is it not a wedge sum/product. I might suggest:

> Appears as Example 1.25 in {{Hatcher}} under the name "Shrinking wedge of circles"; however, note that it is not actually an example of a "wedge sum" as defined in e.g. {{wikipedia:Wedge_sum}}, {{doi:10.1007/978-1-4612-4576-6}}.

File: spaces/S000139/README.md
Comment (StevenClontz) [reply spaces/S000139/README.md:7]: Another option: "Rose with $\omega$-many petals".

**Comment by StevenClontz** [comment]
Marking as draft until @danflapjax has opportunity to consider feedback from reviews.

**Review by StevenClontz — APPROVED** [review]

**Comment by prabau** [comment]
@StevenClontz @danflapjax 

"Rose with countably infinite petals"

Guys, what were you thinking?  As written, it's not what you mean.  It even does not make much sense grammatically.

"A forest with infinite trees" = a forest with trees such the trees (or some of the trees) are infinite.
"A forest with infinitely many trees" = ok

"A forest with countably infinite trees" = a forest with trees such the trees (or some of the trees) are countably infinite, whatever that could mean.
"A forest with countably many trees" = ok

Sorry I did not have the chance to review before it was approved.  I'll propose a fix in a separate PR.

**Comment by StevenClontz** [comment]
I think your comment brings up a valid point, although you did not express it as diplomatically as I'd like to encourage in this community.

But since you asked what I was thinking, the simple answer is that I didn't review that particular line as carefully as I should have, given that this PR was solely for editing titles and descriptions. (If it had a contribution that would fill in gaps in our database, then I'd definitely not want this detail to block merging a PR.) I would still prefer "rose with $\omega$-many petals". I'll make that PR myself unless you make one first.

**Comment by prabau** [comment]
Thanks for the reminder.  I'll be try to be more diplomatic next time. :)

**Pull request merged**
